### PR TITLE
Index only recently updated media

### PR DESCRIPTION
**Pull request by AndreasPB**
# Description
* Gotta go(python) fast

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: backend/cli.py [hunk 1]
```diff
@@ -47,17 +49,24 @@ def update_ids(ids: list[str]):
...
     if get_settings().app_environment == Environment.DEVELOPMENT:
         # Is ran at startup in production
         update_index()
 
+    db = database.SessionLocal()
+
+    if first_time:
+        db_media = get_all_media(db)
+    else:
+        db_media = get_recently_updated_media(db, hours=1)
```
Comment (Primdahl26) [inline backend/cli.py:62]: Should this be more than 1 hour now to be sure? Since we have to do `--first-time` every time?

Comment (AndreasPB) [reply backend/cli.py:62]: This is the else for when it isn't first time :p The other one takes everything

Comment (Primdahl26) [reply backend/cli.py:62]: True lol, but this will basically never be used then right? Other than locally

Comment (AndreasPB) [reply backend/cli.py:62]: Yup, it was made a "long" time ago when we didn't know any better :D But I still think it makes sense for the shorter flow. Might be useful later :shrug:

**Pull request closed without merging**